tests/test_extension_statsmailer.py

```diff
@@ -0,0 +1,82 @@
+from unittest.mock import MagicMock
+
+import pytest
+
+from scrapy import signals
+from scrapy.exceptions import NotConfigured
+from scrapy.extensions import statsmailer
+from scrapy.mail import MailSender
+from scrapy.signalmanager import SignalManager
+from scrapy.statscollectors import StatsCollector
+from scrapy.utils.spider import DefaultSpider
+
+
+@pytest.fixture
+def dummy_stats():
+    class DummyStats(StatsCollector):
+        def __init__(self):
+            # pylint: disable=super-init-not-called
+            self._stats = {"global_item_scraped_count": 42}
+
+        def get_stats(self, *args, **kwargs):
```

Comment:
`args` and `kwargs` aren't used here.

Reply:
Right, the warning for that line wasn't caught because that line wasn't covered by tests. You should remove the spider arg passing from that line.